File: FragmenttoFragment/app/src/main/java/com/example/fragmenttofragment/MainActivity.java
package com.example.fragmenttofragment;

import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.FragmentManager;
import androidx.fragment.app.FragmentTransaction;

import android.os.Bundle;

import com.example.fragmenttofragment.Fragment.FragmentA;
import com.example.fragmenttofragment.Fragment.FragmentB;

public class MainActivity extends AppCompatActivity{
    FragmentA fragmentA;
    FragmentB fragmentB;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        fragmentA = new FragmentA();
        fragmentB = new FragmentB();
        FragmentManager fragmentManager = getSupportFragmentManager();
        FragmentTransaction ft = fragmentManager.beginTransaction();
        ft.replace(R.id.container_a, fragmentA);
        ft.replace(R.id.container_b, fragmentB);
        ft.commit();
    }
//    using implements interface in Fragment class to use function below
//    @Override
//    public void setInputA(String input) {
//        fragmentB.updateInput(input);
//    }
//
//    @Override
//    public void setInputB(String input) {
//        fragmentA.updateInput(input);
//    }
}

File: CoronaAppMobile/app/src/main/java/com/example/coronaapp/app/MainActivity.kt
package com.example.coronaapp.app

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.navigation.fragment.NavHostFragment
import androidx.navigation.ui.setupWithNavController
import com.example.coronaapp.R
import com.google.android.material.bottomnavigation.BottomNavigationView

class MainActivity : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val navHostFragment =
            supportFragmentManager.findFragmentById(R.id.container_fragment) as NavHostFragment
        val navController = navHostFragment.navController
        findViewById<BottomNavigationView>(R.id.bottom_nav)
            .setupWithNavController(navController)
    }
}